svelte-kit-scss/svelte.config.js

```diff
@@ -8,7 +8,9 @@ const config = {
   preprocess: preprocess({ typescript: true, scss: true }),
 
   kit: {
-    adapter: adapter(),
+    adapter: adapter({
+      edge: true,
```

Comment:
Why edge too?

Reply:
Because of a bug in this version of sveltekit adaptor 
![image](https://user-images.githubusercontent.com/49324987/207703653-9d27f309-1d9a-47b5-8ee3-d00e039f936e.png)

for more info https://github.com/sveltejs/kit/issues/6462